app/views/topics/show.html.erb

```diff
@@ -15,8 +15,8 @@
 <div class="row">
   <div class="col-md-8">
     <% @bookmarks.each do |topic_bookmark| %>
-      <% @url = "#{topic_bookmark.url}" %>
-      <h4><a href= <%= "#{@url}" %> > <%= topic_bookmark.url %> </a> 
+      <% url = topic_bookmark.url %>
+      <h4><a href= <%= url %> > <%= url %> </a> 
```

Comment:
View source on the topics/show page and you should see that the href attribute for this anchor doesn't have surrounding quotes. Try this instead of lines 18 and 19:
```html
<h4><%= link_to topic_bookmark.url, topic_bookmark.url %>
```
View source again with this change and make sure the href attribute has the surrounding quote marks, e.g.:

```html
<a href="https://www.google.com/">https://www.google.com/</a>
```